src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers/BinaryFormatterUnsafeDeserializeAnalyzer.cs

```diff
@@ -39,71 +38,81 @@ public override void Initialize(AnalysisContext context)
             context.EnableConcurrentExecution();
             context.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.Analyze | GeneratedCodeAnalysisFlags.ReportDiagnostics);
 
-            context.RegisterSyntaxNodeAction(AnalyzeMemberAccessExpressions, SyntaxKind.SimpleMemberAccessExpression);
+            context.RegisterSyntaxNodeAction(AnalyzeMemberAccessExpressions, CodeAnalysis.CSharp.SyntaxKind.SimpleMemberAccessExpression);
```

Comment:
Why not do an operation based? Then you get all the language agnostic features of operating there. I do not see a need to operate in a syntax level for this - it can be (and should be) done before moving to .net core and both APIs are available in .NET Framework

Reply:
Right now, UA only runs code fixes after migration, though. We should add a second source update step so that we can have one before TFM change and one after.

Reply:
do we leave this as-is for now or do we add a second source update step?